test-harness/tests/poll_api.rs

```diff
@@ -210,7 +210,7 @@ fn write_deadlock() {
     // We make the message to transmit large enough s.t. the "server"
     // is forced to start writing (i.e. echoing) the bytes before
     // having read the entire payload.
-    let msg = vec![1u8; 1024 * 1024];
+    let msg = vec![1u8; 100024 * 100024];
```

Comment:
can we maintain the previous values?
I feel only Increasing the flakiness rate isn't much valuable

Reply:
I didn't mean to merge those changes. Do you happen to know how to test this PR properly?